src/GlobalErrorBoundary.js

```diff
@@ -25,9 +25,9 @@ class GlobalErrorBoundary extends React.Component {
 
   render () {
     const title = msg.globalErrorBoundaryTitle()
-    const descr = msg.globalErrorBoundaryDescription({
-      bugUrl: `<a href='https://github.com/oVirt/ovirt-web-ui/issues'>${msg.gitHub()}</a>`,
-    })
+    const descr = branding.fixedStrings.ISSUES_TRACKER_URL ? msg.globalErrorBoundaryDescription({
```

Comment:
This is a bit awkward.
Like in the About box, if the tracker isn't in branding, the "report issues" bit should be hidden instead of an empty link.

Maybe have a `const showDescription = branding.fixedString.ISSUES_TRACKER URL && .....ISSUES_TRACKER_TEXT` and then the `descr` and the html would be wrapped by that constant.

Reply:
the descr need to be wrapped as well? i cant wrap him inside the return section.
I can check if showDescription exists before creating the descr, but it needs to be before the return statement.
right?